tiledb/sm/misc/parallel_functions.h

```diff
@@ -224,7 +224,11 @@ Status parallel_for(
     const uint64_t subrange_end = begin + fn_iter + task_subrange_len;
     std::function<Status()> bound_fn =
         std::bind(execute_subrange, subrange_start, subrange_end);
+#if 01
```

Comment:
Remove. Same below. FYI, when I develop and want to force a single thread in each thread pool, I modify `config.cc` to set the default concurrency levels to `1`.

Reply:
reverted